gluten-core/src/main/java/io/glutenproject/substrait/rel/LocalFilesNode.java

```diff
@@ -30,12 +30,13 @@
 import java.util.List;
 import java.util.Map;
 
-public class LocalFilesNode implements Serializable {
+public class LocalFilesNode implements ReadSplit, Serializable {
```

Comment:
Can we have a better name instead of `ReadSplit` which can show the inheritance relationship here? maybe `FilesNode` if there is kind of `IceBergFilesNode` / `DeltaFilesNode` exist?

Reply:
I see. Then maybe `SplitInfo` is more suitable in this case and it has corresponding definition like "SplitInfo" or `HiveConnectorSplit` at velox side.

Reply:
@yma11 I think it makes sense. I will change it to `SplitInfo`.